boefjes/boefjes/models.py

```diff
@@ -75,6 +77,24 @@ class Config:
         validate_assignment = True
 
 
+class BoefjeConfig(BaseModel):
+    id: int
+    settings: dict
+    enabled: bool
+    boefje_id: str
+    organisation_id: str
+
+    @computed_field  # type: ignore[prop-decorator]
```

Comment:
I'm not a fan of doing this on every read, when we could (and I think should) do so on write. Storing the hash in a database column also gives us the opportunity to query for other organizations that have the same hash by using an fast and indexed column lookup.

Reply:
This does raise some extra security questions, especially since the settings are encrypted at rest. Do we want to hash the database value, meaning the encrypted settings if encryption has been turned on? Note that with the boefje_id and enabled filters, the result set would at most be the number of organizations, where doing this on write means maintaining a computed field that should be kept in sync with the original column, which is not possible to do within the database since we use application-level encryption.